private/data/i18n/en.json

```diff
@@ -241,22 +410,93 @@
         "createOrLogin": "Create an account, or sign in",
         "guestOrLogin": "Sign in, or checkout as a guest",
         "continueAsGuest": "Continue as guest",
-        "guestMessage": "You can checkout without creating an account. You'll have a chance to create an account later.",
-        "addEmail": "Add email"
```

Comment:
I think we still need this email message

Reply:
i18nKey="checkoutLogin.guestMessage"

not removed? ```      "checkoutLogin": {
        "welcome": "Welcome, it's so good to see you.",
        "loginCompleted": "Account completed",
        "createOrLogin": "Create an account, or sign in",
        "guestOrLogin": "Sign in, or checkout as a guest",
        "continueAsGuest": "Continue as guest",
        "guestMessage": "You can checkout without creating an account. You'll have a chance to create an account later."
      },```

Reply:
(or did you mean addEmail).. Fixing.